src/backend/tests/reimbursement-requests.test.ts

```diff
@@ -625,4 +626,29 @@ describe('Reimbursement Requests', () => {
       expect(newReimbursement).toStrictEqual(reimbursementTransformer(reimbursementMock));
     });
   });
+
+  describe('Edit Vendor Tests', () => {
+    test('Throws error if user isnt an admin or head', async () => {
```

Comment:
Let's update the test name to say if the user isn't an admin or finance leads/head